torch/testing/_internal/common_methods_invocations.py

```diff
@@ -11453,6 +11453,8 @@ def ref_pairwise_distance(input1, input2):
            sample_inputs_func=sample_inputs_legacy_solve,
            check_batched_gradgrad=False,
            supports_forward_ad=True,
+           # See https://github.com/pytorch/pytorch/issues/66357
+           check_batched_forward_grad=False,
```

Comment:
This is weird... still investigating.

Reply:
I figured out why this was the case. This failing started happening after adding the check that batch dimensions are at the front. 

See https://github.com/pytorch/pytorch/pull/68631#discussion_r761365724